src/Tools/Auditor/Report.dfy

```diff
@@ -0,0 +1,125 @@
+include "../../Utils/Library.dfy"
+
+module AuditReport {
+  import opened Utils.Lib.Seq
+
+  //// Data types for report ////
+
+  datatype Tag =
+    | IsGhost
+    | IsSubsetType
+    | IsCallable
+    | HasNoBody // TODO: should all tags be positive?
+    | HasAxiomAttribute
+    | HasExternAttribute
+    | HasVerifyFalseAttribute
+    | HasAssumeInBody // Maybe: (expr: Expression) TODO: :axiom?
```

Comment:
Yes, it would be nice to capture the assumption — even better if we could later on print it.

Reply:
Let's create an issue for this